docs/root/version_history/current.rst

```diff
@@ -26,6 +26,7 @@ Removed Config or Runtime
 
 New Features
 ------------
+* http: added support for :ref:`proxy_status_config <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.proxy_status_config>` for configuring [`Proxy-Status`](https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-proxy-status-08) HTTP response header fields.
```

Comment:
See result in: https://storage.googleapis.com/envoy-pr/4f9da61/docs/version_history/current.html
RST uses a different external link style (https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases).

Should be
```suggestion
* http: added support for :ref:`proxy_status_config <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.proxy_status_config>` for configuring `Proxy-Status <https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-proxy-status-08>`_ HTTP response header fields.
```